pdfbox/src/test/java/org/apache/pdfbox/pdmodel/font/TestFontEmbedding.java

```diff
@@ -371,4 +376,116 @@ void testReuseEmbeddedSubsettedFont() throws IOException
             assertEquals(text1 + " " + text2, extractedText.trim());
         }
     }
+
+    private class TrueTypeEmbedderTester extends TrueTypeEmbedder {
```

Comment:
I really only want to expose isEmbeddingPermitted() and make it reachable in tests.  But it's a non-static method on an abstract class.  I needed to extend the class to get one I could instantiate.